app/apphandlers/gatewayAppHandler.go

```diff
@@ -89,6 +89,20 @@ func (a *gatewayApp) StartRudderCore(ctx context.Context, options *app.Options)
 	}
 	defer gatewayDB.Stop()
 
+	errDB := jobsdb.NewForWrite(
+		"proc_error",
+		jobsdb.WithClearDB(options.ClearDB),
+		jobsdb.WithDSLimit(&a.config.gatewayDSLimit), // TODO: use better config
+		jobsdb.WithSkipMaintenanceErr(config.GetBool("Gateway.jobsDB.skipMaintenanceError", false)),
```

Comment:
```suggestion
		jobsdb.WithSkipMaintenanceErr(config.GetBool("Gateway.jobsDB.skipMaintenanceError", true)),
```

Reply:
Removing this line altogether. Since writer doesn't perform any maintenance. (same as above comment)

Reply:
please see my reply above :)